package.json

```diff
@@ -15,6 +15,7 @@
     "test": "jest"
   },
   "dependencies": {
+    "@babel/runtime": "^7.13.10",
```

Comment:
The project was not compiling for me locally without adding this, but can discuss if this was an issue only on my setup

Reply:
I guess this is fine.  Very strange behaviour though.